### PR TITLE
Add Luis Orbaiceta as a contributor

### PR DESCRIPTION
**Pull request by luisorbaiceta**
#### Checklist

- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by jsumners — APPROVED** [review]
image

File: README.md [hunk 1]
```diff
@@ -254,6 +254,7 @@ Team members are listed in alphabetical order.
 * [__Maksim Sinik__](https://github.com/fox1t), <https://twitter.com/maksimsinik>, <https://www.npmjs.com/~fox1t>
 * [__Manuel Spigolon__](https://github.com/eomm), <https://twitter.com/manueomm>, <https://www.npmjs.com/~eomm>
 * [__James Sumners__](https://github.com/jsumners), <https://twitter.com/jsumners79>, <https://www.npmjs.com/~jsumners>
+* [__Luis Orbaiceta__](https://github.com/luisorbaiceta), <https://twitter.com/luisorbai>, <https://www.npmjs.com/~luisorbaiceta>
```
Comment (jsumners) [inline README.md:257]: This should be in alphabetical order. So this line should be the new line 254.

**Review by zekth — APPROVED** [review]
Welcome

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**